scripts/db_migrator.py

```diff
@@ -882,15 +931,38 @@ def version_4_0_0(self):
         self.stateDB.set(self.stateDB.STATE_DB, 'FAST_RESTART_ENABLE_TABLE|system', 'enable', enable_state)
         self.set_version('version_4_0_1')
         return 'version_4_0_1'
-    
+
+
```

Comment:
remove extra line. follow the single line delimiter pattern between functions

Reply:
Oh I am used the pre-commit checker enforcing 2 lines after functions - guess this repo doesn't have one